src/puppetlabs/puppetdb/http/v4.clj

```diff
@@ -49,15 +48,6 @@
    ["resources" &]
    {:any (resources/resources-app version)}
 
-   ["metrics" &]
-   (moustache/app
-    ["mbeans"]
-    {:get met/list-mbeans}
-
-    ["mbean" & names]
```

Comment:
So here is a thing to consider, there is a ticket open today that expresses the desire to make this all 'mbeans' instead of mbean & mbeans (PDB-200). If its not a hard slog, we could do it now as part of this, but its up to you.